.vortex/installer/src/Utils/JsonManipulator.php

```
@@ -40,6 +40,29 @@ public static function fromFile(string $composer_json): ?self {
+   *   Path to the JSON file.
+   * @param callable $callback
+   *   Callback receiving the manipulator for the file.
+   *
+   * @throws \RuntimeException
+   *   If the file cannot be read or does not contain valid JSON.
+   */
+  public static function updateFile(string $file, callable $callback): void {
+    $instance = self::fromFile($file);
+
+    if (!$instance instanceof self) {
+      throw new \RuntimeException(sprintf('Unable to read a JSON file at "%s".', $file));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '58', column '17'). (undefined)

(MissingImport)